packages/neon-core/src/rpc/Query.ts

```diff
@@ -133,6 +133,11 @@ export interface ValidateAddressResult {
   isvalid: boolean;
 }
 
+export interface UnclaimedGasResult {

```

Comment:
Lets get the name in line with the rest of the interfaces. Top level interfaces can follow the name of the RPC call to make the link very clear.
```suggestion
export interface GetUnclaimedGasResult{

```